recipe/build.sh

```diff
@@ -1,36 +1,37 @@
 #!/bin/bash
 
-mkdir build
+mkdir build || true
```

Comment:
is it fine if we remove the build dir if it exists? I don't trust keeping old ones around

Reply:
For conda builds dirty feature I always do this. New builds are started in folders named down to the current utc second so this isn't an issue